app/models/to_solr_behavior.rb

```diff
@@ -0,0 +1,23 @@
+module ToSolrBehavior
```

Comment:
I think this should be deleted, it seems you have the appropriate one in `concerns/scholars_archive`